src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -642,61 +643,32 @@ const DetailsFormLeft: React.FC<DetailsFormLeftProps> = ({
   const renderShowScheduleToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show Schedule</h4>
-      <label id={"showSchedule"} className="switch">
-        <input
-          type="checkbox"
-          id={"showSchedule"}
-          name={"showSchedule"}
-          ref={register}
-          defaultChecked={DEFAULT_SHOW_SCHEDULE}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler
+        id="showSchedule"
+        forwardedRef={register}
+        defaultToggled={DEFAULT_SHOW_SCHEDULE}
+      />
     </div>
   );
 
   const renderShowGridToggle = () => (
     <div className="toggle-room">
       <h4 className="italic input-header">Show grid layout</h4>
-      <label id={"showGrid"} className="switch">
-        <input
-          type="checkbox"
-          id={"showGrid"}
-          name={"showGrid"}
-          ref={register}
-        />
-        <span className="slider round"></span>
-      </label>
+      <Toggler id="showGrid" forwardedRef={register} />
```

Comment:
There's no need for an `id`, because `Checkbox` nests it's `input` within the `label`, so they are inherently connected without needing the `id`.

```suggestion
      <Toggler name="showGrid" forwardedRef={register} />
```

Reply:
Agree https://github.com/sparkletown/sparkle/pull/1775/commits/3c68fdfc66a9346cd67e1d15bd5fe7a688d52090#r667327574